CodePushConfig.m

```diff
@@ -31,6 +33,9 @@ - (instancetype)init
     NSString *deploymentKey = [infoDictionary objectForKey:@"CodePushDeploymentKey"];
     NSString *serverURL = [infoDictionary objectForKey:@"CodePushServerURL"];
     
+    NSUserDefaults* userDefaults = [NSUserDefaults standardUserDefaults];
```

Comment:
Xcode complains about this variable not being used. Should we cache the UUID in defaults after all? I noticed that is what the Cordova device plugin does, and I wonder if that helps support edge-case scenarios where a new UUID would be generated when upgrading an existing binary installation.